compiledb-gen-parser

```diff
@@ -96,8 +100,26 @@ def parse_flags(build_log, proj_dir, inc_prefix, verbose):
     line_count = 0
     compiler = None
 
+    dir_stack = []
+    sub_dir = proj_dir
```

Comment:
Please let's use a more meaningful name here, maybe `working_dir` or `build_dir` ?

Reply:
@nickdiego Yes, it's better to name it as ```working _dir```.